dashboard/api.py

```diff
@@ -514,28 +510,20 @@ def get_certificate_url(mmtrack, course):
     return url
 
 
-def get_overall_final_grade_for_course(mmtrack, course):
+def get_overall_final_grade_for_course(user, course):
     """
     Calculate overall grade for course
 
     Args:
-        mmtrack (dashboard.utils.MMTrack): an instance of all user information about a program
+        user (django.contrib.auth.models.User): A user
         course (courses.models.Course): A course
     Returns:
         str: the overall final grade
     """
-    final_grades = mmtrack.get_passing_final_grades_for_course(course)
-    best_grade = final_grades.first()
-    if best_grade is None:
-        return ""
-    if not course.has_exam:
-        return str(round(best_grade.grade_percent))
-
-    best_exam = mmtrack.get_best_proctored_exam_grade(course)
-    if best_exam is None:
-        return ""
-
-    return str(round(best_grade.grade_percent * COURSE_GRADE_WEIGHT + best_exam.score * EXAM_GRADE_WEIGHT))
+    combined_grade = CombinedFinalGrade.objects.filter(user=user, course=course)
```

Comment:
So in testing this, I realized that this is going to cause combined final grades to disappear until we run the populate command. That might be why you split the two PRs in the first place. So I'd suggest that we just feature flag the old code vs. new code so that we continue to use the old code (removed in this diff) until we run the command, then we can set the flag to `True` and switch to looking up `CombinedFinalGrade` and finally follow this up with a tech debt issue to remove the feature flag and old code. Does that make sense?